chucks-pick3-web/angular-app/src/app/app.module.ts

```diff
@@ -21,6 +21,7 @@ import {InfoInterfaceComponent} from './components/info-interface.component/info
 import {InfoGenerateComponent} from './components/info-generate.component/info-generate.component';
 import {InfoRequirementsComponent} from './components/info-requirements.component/info-requirements.component';
 import { ClickOutsideModule } from 'ng-click-outside';
+import {ScrollEventModule} from 'ngx-scroll-event';
```

Comment:
Nice Job on finding this module.  Be sure to add a space between curly braces and Module name.